ogr/abstract.py

```diff
@@ -858,6 +866,16 @@ def can_merge_pr(self, username: str) -> bool:
         """
         raise NotImplementedError()
 
+    def add_user(self, user, access_level: AccessLevel) -> None:
```

Comment:
Please, use type annotations for `user` as well and also in the methods with the implementation.